docs/fastdds/property_policies/ignore_local_endpoints.rst

```diff
@@ -15,7 +15,17 @@ from a |DataWriter| belonging to the same |DomainParticipant| on the |DataReader
 |GuidPrefix_t-api|), this entails for a data sample to go all the way to the |DataReaderListener| just to be discarded
 by an overcomplicated application business logic.
 For this reason, Fast DDS offers the possibility of instructing the |DomainParticipant| to avoid the matching of local
-endpoints through the following property:
+endpoints through the PropertyPolicyQos.
+The property can be configured both at the DomainParticipant level as at the endpoint level:
+
+* When configured in the DomainParticipant, no local endpoint will be matched.
+* When configured in a specific endpoint, that specific endpoint will not be matched with any local one, but any other
```

Comment:
Probably the clarification regarding other endpoints is superfluous since we are already specifying that the behavior relates to this specific endpoint.

Reply:
I have reword the sentence. I agree what you say, but what I am trying to communicate is the possible asymmetry of one endpoint with the property set and another one with the property not set.